pytorch_to_returnn/converter/converter.py

```diff
@@ -248,17 +303,19 @@ def _run_returnn_standalone_net_dict(self):
     with make_scope() as session:
       from returnn.config import Config
       from returnn.tf.network import TFNetwork
+      extern_data = {
+        str(i) if i else "data": dict({"available_for_inference": True}, **v)
+        for i, v in enumerate(self._returnn_in_data_dict.values())}
```

Comment:
The order of `values` is non-deterministic in older Python versions. I'm not sure if this is good then.